### DISCUSSION TITLE
실시간조회등록에 종목별프로그램매매 관련 FID가 등록되지 않습니다.

### DISCUSSION THREAD

**Discussion by Sanghoon-ai** [elbakramer/koapy#67] [Q&A]
안녕하세요!

KOAPY 로 Setrealreg 와 동등한 entrypoint.GetRealDataForCodesAsStream 로 요청한 FID 값이 실시간으로 나오지 않고
'주식체결' 의   Real Type 의 FID값만 등록되어 실시간으로 종목별프로그램매매 관련 FID 값을 받을수 없습니다.

![image](https://user-images.githubusercontent.com/65003780/156977755-bba61242-e24f-453c-a1cc-bf0e574b7846.png)

fid_list = KiwoomOpenApiPlusRealType.get_fids_by_realtype_name('종목별프로그램매매') 

이렇게 하면 FID에 종목별프로그램매매 관련 FID 값이 들어가서 실시간 요청이 되어야 할것 같은데 

PPRINT 하면 '주식체결' 관련 FID 값만 나옵니다. 

확인 가능할까요? 

##########################
자답을 하네요 키움 계시판에 따로 나오네요!  감사합니다. 좋은 프로그램 만들어 주셔서! ^^

**Comment by dh377** [comment]
해당 기능은 무효화 되었습니다. 작년부터 OpenAPI+ 스팩이 바뀌어서 요청한대로 FID를 리턴하지 않게 되었기 때문입니다. 패킷이 실시간 수신 될 때, 키움측에서 설정한 `실시간 타입`별로 미리 정의된 FID로만 항상 리턴됩니다.
